### PR TITLE
Update Stable Diffusion notebook for 0.2.3

### PR DESCRIPTION
**Pull request by pcuenca**
# What does this PR do?

Update Stable Diffusion colab for diffusers release 0.2.3.

## Who can review?

@patrickvonplaten

### CODE REVIEW

File: diffusers/stable_diffusion.ipynb [hunk 1]
```diff
@@ -12,7 +12,7 @@
...
         "See the [model card](https://huggingface.co/CompVis/stable-diffusion) for more information.\n",
         "\n",
-        "**The Stable Diffusion weights are currently only available to universities, academics, research institutions and independent researchers. Please request access applying to <a href=\"https://stability.ai/academia-access-form\" target=\"_blank\">this</a> form**\n",
+        "You need to accept the model license before downloading or using the weights. Please, visit the [model card](https://huggingface.co/CompVis/stable-diffusion), read the license and tick the checkbox if you agree. You have to be a registered user in 🤗 Hugging Face Hub, and you'll also need to use an access token for the code in this notebook to work. For more information on access tokens, please refer to [this section of the documentation](https://huggingface.co/docs/hub/security-tokens).\n",
```
Comment (pcuenca) [inline diffusers/stable_diffusion.ipynb:15]:
```suggestion
"You need to accept the model license before downloading or using the weights. Please, visit the [model card](https://huggingface.co/CompVis/stable-diffusion-v1-4), read the license and tick the checkbox if you agree. You have to be a registered user in 🤗 Hugging Face Hub, and you'll also need to use an access token for the code in this notebook to work. For more information on access tokens, please refer to [this section of the documentation](https://huggingface.co/docs/hub/security-tokens).\n",
```

File: diffusers/stable_diffusion.ipynb [hunk 2]
```diff
@@ -361,7 +359,7 @@
         "from diffusers import StableDiffusionPipeline\n",
         "\n",
         "# make sure you're logged in with `huggingface-cli login`\n",
-        "pipe = StableDiffusionPipeline.from_pretrained(\"CompVis/stable-diffusion-v1-3-diffusers\", use_auth_token=True)  "
+        "pipe = StableDiffusionPipeline.from_pretrained(\"CompVis/stable-diffusion-v1-3\", use_auth_token=True)  "
```
Comment (pcuenca) [inline diffusers/stable_diffusion.ipynb:362]:
```suggestion
        "pipe = StableDiffusionPipeline.from_pretrained(\"CompVis/stable-diffusion-v1-4\", use_auth_token=True)  "
```

**Review by pcuenca — COMMENTED** [review]
Updated model to v1-4

**Comment by patrickvonplaten** [comment]
Sorry I think there is a conflict now - very sorry to review this late @pcuenca . Also cc @anton-l 

Should we try to rebase the PR?

**Comment by anton-l** [comment]
Closing as stale

**Pull request closed without merging**